src/gtfs/read.rs

```diff
@@ -79,6 +79,7 @@ impl From<Agency> for objects::Company {
             url: Some(agency.url),
             mail: agency.email,
             phone: agency.phone,
+            codes: BTreeSet::new(),
```

Comment:
`..Default::default()` maybe ?
And same thing in tests in in `/src/ntfs/mods.rs`

Reply:
In From trait, I prefer it to be explicit.
in the tests why not @woshilapin ?

Reply:
It's just a detail. PR is ready to go :wink:

Reply:
For one field, I'd prefer it to be explicit.

Reply:
Ok.So go !